File: trapezoid_area/trapezoid_area.cpp
// input - procceses - output

// (a,b,h) - (.5*(a+b)*h) - area

// input - procceses - output

// (m,h) - (m*h) - area


#include <iostream>
using namespace std;

float trape_area(float a,float b,float h){
  return 0.5 * (a+b) * h;
}

int main(){
  float a,b,h, area;
  cout << "enter the first base a = ";
  cin >> a;
  cout << "enter the 2nd base b = ";
  cin >> b;
  cout << "enter the height h = ";
  cin >> h;
  area=trape_area(a,b,h);
  cout << "area = h * (a+b) / 2\n";
  cout << "     = " << area << "\n";
  return 0;
}

File: trapezoid_area/trapezoid_area.py
def calc_area(a: float, b: float, h: float):
  return 0.5*(a+b)*h

if __name__== "__main__":
  a=float(input('enter the first base a = '))
  b=float(input('enter the 2nd base b = '))
  h=float(input('enter the height h = '))
  area=calc_area(a,b,h)
  print(f'area = h * (a+b) / 2\n',f'    = {area}')
  